service/src/main/java/bio/terra/workspace/db/model/DbWorkspaceActivityLog.java

```diff
@@ -0,0 +1,23 @@
+package bio.terra.workspace.db.model;
```

Comment:
optional

Hmm I think there was an earlier thread where you created this so you don't have to pass in a bunch of parameters. But this just adds an extra step for callers. Callers have to list parameters regardless, and with DbWorkspaceActivityLog, they have to create a DbWorkspaceActivityLog. 

Without DbWorkspaceActivityLog:

Every caller passes in changeType

With DbWorkspaceActivityLog

Every caller creates DbWorkspaceActivityLog, passes in changeType
Every caller calls writeActivity with DbWorkspaceActivityLog

Removing DbWorkspaceActivityLog would also make the code easier to read.

With:

        workspaceActivityLogDao.writeActivity(
            workspaceUuid, new DbWorkspaceActivityLog().operationType(OperationType.UPDATE));

Without:

        workspaceActivityLogDao.writeActivity(workspaceUuid, OperationType.UPDATE);

If you keep DbWorkspaceActivityLog, can you take out validating OperationType, since that's validated in writeActivity()?

(Also, on line 21 it's weird to create an Optional that isn't used. You could just throw if operationType isn't set.)